packages/hosting/src/api/zonke.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    SupportedFrameworks,
+    createPreviewEnvironment,
+    deployToPreviewEnvironment,
+    getPreviewEnvironment,
+} from '@zonke-cloud/sdk';
+
+export interface CreateEnvOptions {
+    userId: string;
+    framework: 'nextjs' | 'remix' | 'react';
+}
+
+export async function createEnv(options: CreateEnvOptions) {
+    const framework = options.framework as SupportedFrameworks;
+    const awsHostedZone = 'zonke.market';
+
+    return createPreviewEnvironment({
+        userId: options.userId,
+        framework,
+        awsHostedZone,
+    });
+}
+
+export interface DeployOptions {
+    environmentId: string;
+    buildOutputDirectory: string;
+    message?: string;
+}
+
+export async function deploy(options: DeployOptions) {
+    const { environmentId, buildOutputDirectory, message } = options;
+
+    return deployToPreviewEnvironment({
+        environmentId,
+        buildOutputDirectory,
+        message,
+        // Default to 60 seconds for upload link expiration
+        uploadLinkExpirationOverride: 60,
+    });
+}
```

Comment:
I had to add a `packageJsonPath` property to support Remix builds. The default build output for Remix does not include the required modules, so we need to copy the `package.json` path so dependencies can be installed during deployment. See this CLI implementation:

https://github.com/zonke-inc/sdk-ts/blob/main/packages/cli/src/commands.ts#L37-L43